web/src/components/Meditor/Meditor.vue

```diff
@@ -283,16 +283,10 @@ const invalidPermissionSnackbar = ref(false);
 const tab = ref(null);
 // const loadFromLocalStoragePrompt = ref(false);
 
-editorInterface.value = new EditorInterface(schema.value, model.value as DandiModel);
-const {
-  modelValid,
-  basicSchema,
-  basicModel,
-  basicModelValid,
-  complexSchema,
-  complexModelValidation,
-  transactionTracker,
-} = editorInterface.value;
+watchEffect(() => {
+  editorInterface.value = new EditorInterface(schema.value, model.value as DandiModel);
+});
```

Comment:
This is the crux of the fix, right? We stop using a single, one-time-defined instance of `EditorInterface` whose components were also statically "exported" to the rest of the module, and instead refresh it every time stuff changes under us?

A note explaining this in the PR description would be helpful to our future selves.

Reply:
Yes, that is the crux of the fix. But, doing that broke a few other things that I am currently debugging.

Reply:
I updated the description, and also changed the approach to fixing the issue (see the PR description)